_posts/2022-09-21-#25880.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: pr
+date: 2022-09-21
+title: "Make stalling timeout adaptive during IBD"
+pr: 25880
+authors: [mzumsande]
+components: ["p2p"]
+host: glozow
+status: upcoming
+commit: 48e5385
+---
+
+## Notes
+
+- During block download, nodes have a `BLOCK_DOWNLOAD_WINDOW=1024`-block window within their best
+  chain (this happens after downloading the full headers chain) and download those blocks in
+parallel.
```

Comment:
From the PR description, it looks like this is only during IBD, not block download generally? Also added link and some rephrasing.
```suggestion
- During Initial Block Download (IBD), after downloading the full headers chain, nodes have a [`BLOCK_DOWNLOAD_WINDOW=1024`](https://github.com/bitcoin/bitcoin/blob/a688ff9046a9df58a373086445ab5796cccf9dd3/src/net_processing.cpp#L128)-block window within their best
  chain during which blocks are downloaded in parallel.
```